ChangeLog

```diff
@@ -4,6 +4,7 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 Features
    * Allow comments in test data files.
+   * Add support for public keys encoded in PKCS#1 format
```

Comment:
Style niggle: Could you please add a dot at the end?

Reply:
done